2.4/examples/replica/mongodb-clustered.json

```diff
@@ -94,7 +94,13 @@
                 "command": [
                   "run-mongod", "initiate"
                 ],
-                "containerName": "mongodb"
+                "containerName": "mongodb",
+                "env": [
+                  {
+                    "name": "MONGODB_REPLICA_NODES",
+                    "value": "3"
```

Comment:
@bparees [Here](https://github.com/openshift/mongodb/pull/143/files/b43f319ed916a1c9c108b349971b90b52dc87c23#diff-25d0fbd40ccee5d2d449d8824f9038c9L107) we're specifying desired number of replicas.